app.js

```diff
@@ -0,0 +1,59 @@
+const taskManager = () => {
+  let todos = [];
+  
+  const $todoInput = document.getElementById("inputTodo");
+  const $addTodoButton = document.getElementById("addTodoButton");
+  const $todoList = document.getElementById("listTodo");
+  const $feedbackMessage = document.getElementById("feedbackMessage");
+
+  function renderTodos() {
+    $todoList.innerHTML = "";
+
+    todos.forEach((todo) => {
+      const todoElement = createTodo(todo);
+      $todoList.appendChild(todoElement);
+    });
+  }
+
+  function createTodo(todo) {
+    const $li = document.createElement("li");
+    const $todoText = document.createElement("span");
+    $todoText.textContent = todo.title;
+
+    $li.appendChild($todoText);
+    return $li;
+  }
+
+  function showFeeback(content) {
+    $feedbackMessage.innerText = content;
+    $feedbackMessage.style.color = "red";
+    $feedbackMessage.style.fontSize = "12px";
+  }
+
+  function clearFeedback() {
+    $feedbackMessage.innerText = "";
+  }
```

Comment:
Move these files to `utility.js` file and import it from there

Reply:
Updated the code in utility.js file